scripts/linters/pylint_extensions_test.py

```diff
@@ -3340,3 +3340,183 @@ def __init__(self, fake_arg):
 
         with checker_test_object.assertNoMessages():
             checker_test_object.checker.visit_classdef(metaclass_node)
+
+
+class DisallowHandlerWithoutSchemaTests(unittest.TestCase):
+
+    def setUp(self):
+        super(DisallowHandlerWithoutSchemaTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.DisallowHandlerWithoutSchema)
+        self.checker_test_object.setup_method()
+
+    def test_schema_handlers_without_request_args_raise_error(self):
+
+        schemaless_class_node = astroid.extract_node(
+            """
+            class BaseHandler():
+                HANDLER_ARGS_SCHEMAS = None
+                URL_PATH_ARGS_SCHEMAS = None
+
+            class FakeClass(BaseHandler):
+                URL_PATH_ARGS_SCHEMAS = {}
+            """)
+        with self.checker_test_object.assertAddsMessages(
+            testutils.Message(
+                msg_id='no-schema-for-handler-args',
+                node=schemaless_class_node,
+                args=(schemaless_class_node.name)
+            )
+        ):
+            self.checker_test_object.checker.visit_classdef(
+                schemaless_class_node)
+
+    def test_schema_handlers_without_url_path_args_raise_error(self):
+
+        schemaless_class_node = astroid.extract_node(
+            """
+            class BaseHandler():
+                HANDLER_ARGS_SCHEMAS = None
+                URL_PATH_ARGS_SCHEMAS = None
+
+            class FakeClass(BaseHandler):
+                HANDLER_ARGS_SCHEMAS = {}
+            """)
+
+        with self.checker_test_object.assertAddsMessages(
+            testutils.Message(
+                msg_id='no-schema-for-url-path-elements',
+                node=schemaless_class_node,
+                args=(schemaless_class_node.name)
+            )
+        ):
+            self.checker_test_object.checker.visit_classdef(
+                schemaless_class_node)
+
+    def test_handlers_with_schema_do_not_raise_error(self):
```

Comment:
`test_handlers_with_valid_schema_do_not_raise_error ` (Asking this to rename because it's representing the test where the schema is present and is correct - note we have errors for them being absent and present but incorrect)

Reply:
Done